<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="11155" started-at="2018-12-19T04:11:04Z" finished-at="2018-12-19T04:11:15Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="11155" started-at="2018-12-19T04:11:04Z" finished-at="2018-12-19T04:11:15Z">
      <class name="testSuite.Client">
        <test-method status="PASS" signature="test1()[pri:0, instance:testSuite.Client@363f0ba0]" name="test1" duration-ms="10718" started-at="2018-12-18T23:11:04Z" finished-at="2018-12-18T23:11:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="test2()[pri:0, instance:testSuite.Client@363f0ba0]" name="test2" duration-ms="83" started-at="2018-12-18T23:11:14Z" finished-at="2018-12-18T23:11:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="test3()[pri:0, instance:testSuite.Client@363f0ba0]" name="test3" duration-ms="127" started-at="2018-12-18T23:11:14Z" finished-at="2018-12-18T23:11:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="PASS" signature="test4()[pri:0, instance:testSuite.Client@363f0ba0]" name="test4" duration-ms="78" started-at="2018-12-18T23:11:15Z" finished-at="2018-12-18T23:11:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test4 -->
        <test-method status="FAIL" signature="testO()[pri:0, instance:testSuite.Client@363f0ba0]" name="testO" duration-ms="64" started-at="2018-12-18T23:11:15Z" finished-at="2018-12-18T23:11:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<401> but was:<200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<401> but was:<200>at org.testng.AssertJUnit.fail(AssertJUnit.java:59)at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:245)at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:252)at testSuite.Client.testO(Client.java:20)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testO -->
      </class> <!-- testSuite.Client -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
